src/analysis/retail/priest/holy/modules/tww/HolyTWW4pcS1.tsx

```diff
@@ -0,0 +1,75 @@
+import Analyzer from 'parser/core/Analyzer';
+import { CastEvent, GetRelatedEvents, HealEvent } from 'parser/core/Events';
+import { SPELLS_THAT_PROC_S1_4PC_HOLY_ID } from '../../constants';
+import { HOLY_TWW_S1_4PC } from '../../normalizers/CastLinkNormalizer';
+import HolyWordCDR from '../core/HolyWordCDR';
+import HIT_TYPES from 'game/HIT_TYPES';
+
+class HolyTWW4pS1 extends Analyzer {
+  static dependencies = {
+    holyWordCDR: HolyWordCDR,
+  };
+  protected holyWordCDR!: HolyWordCDR;
+
+  active4p = false;
+
+  // If a heal event hits the same person twice then its a tier proc
+  detect4pcProc(event: CastEvent): boolean {
```

Comment:
```suggestion
  is4pcProc(event: CastEvent): boolean {
```
nit: naming, prefer using a binary word instead of the vague `detect`